community/teams/moderation.tt

```diff
@@ -1,36 +1,41 @@
-[% WRAPPER layout.tt title="Discourse Team" handlesLayout=1 %]
+[% WRAPPER layout.tt title="Moderation Team" handlesLayout=1 %]
 
 <div class="page-title">
   <div>
     <a href="[% root%]community/index.html#governance-teams">Community</a>
     <span>→</span>
   </div>
-  <h1>Discourse Team</h1>
+  <h1>Moderation Team</h1>
 </div>
 
 <section class="governance-description">
-  This team is responsible for configuring and moderating the
-  <a href="https://discourse.nixos.org/">Discourse forum</a>.
+  This team is responsible for moderating participation on
+official community platforms in accord with <a
+href="https://github.com/NixOS/rfcs/blob/master/rfcs/0102-moderation-team.md">RFC
+102</a>.
 </section>
 
 <section class="governance-team">
 
   <aside>
     <h2>Members</h2>
     <ul>
-      <li><a href="https://discourse.nixos.org/u/zimbatm">zimbatm</a> (zimbatm)</li>
-      <li><a href="https://discourse.nixos.org/u/ryantm/">Ryan Mulligan</a> (ryantm)</li>
-      <li><a href="https://discourse.nixos.org/u/garbas">Rok Garbas</a> (garbas)</li>
+      <li><a href="https://discourse.nixos.org/u/domenkozar/">Domen Kožar</a> (domenkozar)</li>
       <li><a href="https://discourse.nixos.org/u/grahamc">Graham Christensen</a> (gchristensen)</li>
       <li><a href="https://discourse.nixos.org/u/mic92">Jörg Thalheim</a> (Mic92)</li>
+      <li><a href="https://discourse.nixos.org/u/garbas">Rok Garbas</a> (garbas)</li>
+      <li><a href="https://discourse.nixos.org/u/ryantm/">Ryan Mulligan</a> (ryantm)</li>
+      <li><a href="https://discourse.nixos.org/u/zimbatm">zimbatm</a> (zimbatm)</li>
     </ul>
   </aside>
 
   <h2>Reporting issues</h2>
-  <p>
-    If you have an issue with the forum or its content, you can send a direct
-    message on the forum to any of these people.
-  </p>
+
+  <p>Instances of abusive, harassing, or otherwise unacceptable behavior may be
+    reported by contacting the project team at
+    <a href="mailto:community@nixos.org">community@nixos.org</a>.
+    The project team is obligated to maintain confidentiality with regard to the
```

Comment:
```suggestion
    The moderation team is obligated to maintain confidentiality with regard to the
```

Reply:
I think this suggested change might be inaccurate, since I do not know who has access to that email address. I don't for example. I copied that from the top of the Community page.

Reply:
It goes to me and possibly someone at the foundation. I think there is good utility in having the moderation team be contacted collectively, and perhaps an escalation point at the foundation.

Reply:
For now we can probably just leave it as is.

Reply:
Would you like me to add others from the moderation team to the `community@nixos.org` email?